lti_consumer/utils.py

```diff
@@ -76,3 +76,17 @@ def get_lti_ags_lineitems_url(lti_config_id):
         lms_base=get_lms_base(),
         lti_config_id=str(lti_config_id),
     )
+
+
+def get_lti_ags_lineitem_url(lti_config_id, lineitem_id):
```

Comment:
Instead of this, you can re-use the function above and add an optional parameter `lineitem_id` which appends `/` + the id at the end of the URL.